spec/models/piece_spec.rb

```diff
@@ -1,6 +1,33 @@
 require 'rails_helper'
 
 RSpec.describe Piece, type: :model do
+  describe 'move_to! method' do
+    let(:game) { FactoryGirl.create(:game) }
+    it 'should return error if target piece has same color as current piece' do
+      current_piece = FactoryGirl.create(:piece, color: 0, game_id: game.id)
+      target_piece = FactoryGirl.create(:piece, x_coordinate: 4, y_coordinate: 4, color: 0, game_id: game.id)
+      expect(current_piece.move_to!(4, 4)).to eq('Error')
+    end
+
+    it 'should capture if target piece has the opposite color as current piece' do
+      current_piece = FactoryGirl.create(:piece, color: 0, game_id: game.id)
+      target_piece = FactoryGirl.create(:piece, x_coordinate: 4, y_coordinate: 4, color: 1, game_id: game.id)
+      current_piece.move_to!(4, 4)
+      expect(game.pieces.size).to eq(33)
+    end
+
+    it 'should update to the new_x, new_y coordinates (no piece at target)' do
+      FactoryGirl.create(:piece, color: 0, game_id: game.id).move_to!(4, 4)
+      expect(game.pieces.find_by(x_coordinate: 4, y_coordinate: 4) != nil).to eq(true)
```

Comment:
rubcop is complaining because this reads as "expect piece to not be nil". (double negative)

write this instead "expect piece to be present"

`expect(game.pieces.find_by(x_coordinate: 4, y_coordinate: 4)).to be_present`